chia/wallet/wallet_transaction_store.py

```diff
@@ -19,17 +19,14 @@ class WalletTransactionStore:
 
     db_connection: aiosqlite.Connection
     db_wrapper: DBWrapper
-    cache_size: uint32
     tx_record_cache: Dict[bytes32, TransactionRecord]
-    tx_wallet_cache: Dict[int, Dict[Any, Set[bytes32]]]
     tx_submitted: Dict[bytes32, Tuple[int, int]]  # tx_id: [time submitted: count]
     unconfirmed_for_wallet: Dict[int, Dict[bytes32, TransactionRecord]]
 
     @classmethod
-    async def create(cls, db_wrapper: DBWrapper, cache_size: uint32 = uint32(600000)):
+    async def create(cls, db_wrapper: DBWrapper):
         self = cls()
 
-        self.cache_size = cache_size
```

Comment:
is this part of the patch effectively making the cache unlimited?

Reply:
yes